community/cypher/cypher/src/test/scala/org/neo4j/cypher/SemanticErrorAcceptanceTest.scala

```diff
@@ -666,6 +666,18 @@ class SemanticErrorAcceptanceTest extends ExecutionEngineFunSuite {
       "A single relationship type must be specified for MERGE (line 1, column 9 (offset: 8))")
   }
 
+  test("give a nice error message when missing crs in cartesian point") {
+    executeAndEnsureError("RETURN point({x: 2.3, y: 4.5}) as point",
+                          "A cartesian point must contain a 'crs' (coordinate reference system) (line 1, column 14 (offset: 13))")
```

Comment:
I don't understand the error. Without a CRS, this will get WGS84 which makes it a geographic point, not a cartesian point. Then the error should be 'A geographic point needs latitude and longitude', right?